src/legacy/core_plugins/metrics/public/setup.ts

```diff
@@ -19,17 +19,18 @@
 
 // @ts-ignore
 import { functionsRegistry } from 'plugins/interpreter/registries';
+import { CoreSetup, PluginsServiceSetup } from 'src/core/server';
```

Comment:
We should be importing from `core/public` here since this is our client-side shim.

Reply:
Silly me. Of course. Thanks. Changed